src/components/App/App.jsx

```diff
@@ -42,12 +43,15 @@ export default function App() {
 
   const sampleRecipeInfo = constRecipeInfo
 
+  const api_url = "http://localhost:3001/apirecipes/"
+
 // state variables
 const [products, setProducts] = useState(basicProducts)
 const [productForm, setProductForm] = useState(basicProductForm)
-const [recipeModelShow, setRecipeModalShow] = useState(false);
-const [recipeInstructions, setRecipeInstructions] = useState([]);
-const [recipeinfo, setRecipeinfo] = useState({});
+const [recipeModelShow, setRecipeModalShow] = useState(false)
+const [recipeInstructions, setRecipeInstructions] = useState([])
+const [recipeinfo, setRecipeinfo] = useState({})
```

Comment:
I initially had it as `recipeInfo` and `setRecipeInfo` but the console.log was showing warnings that `recipeInfo` suggests the variable to be a DOM element, so I changed the capitalization.
<img width="330" alt="Screen Shot 2022-07-08 at 2 53 15 PM" src="https://user-images.githubusercontent.com/62860690/178075123-634a7b5f-8a09-486e-a6cf-a815a0e7d391.png">
I admit that was a lazy fix so I researched what the error meant, and it seems that it's because I pass in all the `props` in RecipeModal.jsx in the `Modal` component, and `recipeInfo` is not a valid attribute to a `Modal` component. I found a fix, which is including this line:
`const {recipeInfo, instructions, title, ...modalProps} = props`
and pass in `{...modalProps}` in the `Modal` component